fe2-android/app/src/test/java/com/github/dedis/student20_pop/utility/handler/MessageHandlerTest.java

```diff
@@ -0,0 +1,304 @@
+package com.github.dedis.student20_pop.utility.handler;
```

Comment:
Not necessary for this PR, but keep this in mind as-needed : 

Do not hesitate to split this up by tape of message handlers, even if you keep the exact same code.
i.e. `LaoHandlerTest`,  `RollCallHandler`, etc. even if you still use the `MessageHandler` entry point.

Also, for now, these handlers depend on the `LaoRepository` for the input, but maybe we'll later be able to split them up as follows:
1. Verify that LaoRepository calls generic message handler on an incoming message
2. Verify that generic message handler calls the right message handler for any given message
3. Verify that the specific message handler processes a given message correctly, mocking the LaoRepository
This would effectively move the tests from an integration-level closer to a unit-level.

Again, no need to do this now, it's not the highest priority :)